examples/src/write.c

```diff
@@ -249,8 +275,15 @@ int main(int argc, char* argv[])
                     "Number of processes:       %d\n"
                     "Each process wrote:        %.2lf MiB\n"
                     "Total data written:        %.2lf MiB\n"
+                    "File create time:          %.6lf sec\n"
                     "Maximum write time:        %.6lf sec\n"
+                    "Global write time:         %.6lf sec\n"
                     "Maximum sync time:         %.6lf sec\n"
+                    "Global sync time:          %.6lf sec\n"
+                    "File close time:           %.6lf sec\n"
+                    "File laminate time:        %.6lf sec\n"
+                    "Full write time:           %.6lf sec\n"
+                    "File stat time:            %.6lf sec\n"
                     "Aggregate write bandwidth: %.3lf MiB/s\n"
                     "Effective write bandwidth: %.3lf MiB/s\n",
                     io_pattern_str(test_config.io_pattern),
```

Comment:
7 new lines get added to the output. The write CI tests will need to be updated:
Here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/110-write-tests.sh#L71
and here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/110-write-tests.sh#L89